pkg/crc/preflight/preflight_linux.go

```diff
@@ -295,3 +282,10 @@ func distro() *linux.OsRelease {
 	}
 	return distro
 }
+
+func resolvectlExists() bool {
+	if _, err := os.Stat("/usr/bin/resolvectl"); err != nil {
```

Comment:
Instead of hard coding the abs path of `resolvectl` shouldn't just use `exec.LookPath("resolvectl")` ?

Reply:
This function is no longer being used since switching away from `resolvectl` in favor of `systemd-resolve`.